src/SonarQube.Client.Tests/SonarQubeService_GetSuppressedIssuesAsync.cs

```diff
@@ -391,23 +391,52 @@ public async Task GetSuppressedIssuesAsync_From_7_20_BranchIsSpecified_BranchInc
             messageHandler.Reset();
 
             SetupHttpRequest(messageHandler, EmptyGetIssuesResponse);
-            _ = await service.GetSuppressedIssuesAsync("any", "aBranch", CancellationToken.None);
+            _ = await service.GetSuppressedIssuesAsync("any", "aBranch", null, CancellationToken.None);
 
             // The wrapper is expected to make three calls, for code smells, bugs, then vulnerabilities
             var actualRequests = messageHandler.GetSendAsyncRequests();
             actualRequests.Should().HaveCount(3);
-
             actualRequests.Should().OnlyContain(x => x.RequestUri.Query.Contains("&branch=aBranch&"));
         }
 
+        [TestMethod]
+        public async Task GetSuppressedIssuesAsync_From_7_20_IssueKeysAreNotSpecified_IssueKeysAreNotIncludedInQueryString()
+        {
+            await ConnectToSonarQube("7.2.0.0");
+            messageHandler.Reset();
+
+            SetupHttpRequest(messageHandler, EmptyGetIssuesResponse);
+            _ = await service.GetSuppressedIssuesAsync("any", null, null, CancellationToken.None);
+
+            // The wrapper is expected to make three calls, for code smells, bugs, then vulnerabilities
+            var actualRequests = messageHandler.GetSendAsyncRequests();
+            actualRequests.Should().HaveCount(3);
+            actualRequests.Any(x => x.RequestUri.Query.Contains("issues")).Should().BeFalse();
+        }
+
+        [TestMethod]
+        public async Task GetSuppressedIssuesAsync_From_7_20_IssueKeysAreSpecified_IssueKeysAreIncludedInQueryString()
+        {
+            await ConnectToSonarQube("7.2.0.0");
+            messageHandler.Reset();
+
+            SetupHttpRequest(messageHandler, EmptyGetIssuesResponse);
+            _ = await service.GetSuppressedIssuesAsync("any", null, new[]{"issue1", "issue2"}, CancellationToken.None);
+
+            // The wrapper is expected to make three calls, for code smells, bugs, then vulnerabilities
+            var actualRequests = messageHandler.GetSendAsyncRequests();
+            actualRequests.Should().HaveCount(3);
+            actualRequests.All(x => x.RequestUri.Query.Contains("issues=issue1%2Cissue2")).Should().BeTrue();
```

Comment:
We shouldn't need to make three calls if we're fetching a known list of issues. Does it make sense from a SQ point of view to combine `issues` with other filters? If it works then I guess it's ok, but it seems a bit weird.

Reply:
Hmm, so what you're saying is, we can do something like
```
if issueKeys
    make 1 call with branch + issue keys
else
    make 3 calls with branch + type
```

Reply:
I would have thought so. I'm not sure how SQ handles combining the parameters.
The simplistic approach at the ISonarQubeService level would be to have a separate `GetSpecificIssues` that just branch/branch+issuesKeys (whatever is required)